src/app/utils/utils.service.rUtils.js

```diff
@@ -0,0 +1,118 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('roboconf.utils')
+        .service('rUtils', rUtils);
+
+    rUtils.$inject = [ '$timeout' ];
+    
+    /* @ngInject */
+    function rUtils( $timeout ) {
+        var service = {
+            endsWith: endsWith,
+            startsWith: startsWith,
+            isObj: isObj,
+            sortInstances: sortInstances,
+            findPosition: findPosition,
+            showRightBlock: showRightBlock,
+            hideRightBlock: hideRightBlock
+        };
+
+        return service;
+        /////////////////////
+
+        function startsWith( string, prefix ) {
+        	return isObj( string ) && isObj( prefix ) && string.indexOf( prefix ) === 0;
+        }
+
+        function endsWith( string, suffix ) {
+        	return isObj( string ) && isObj( suffix ) && string.indexOf( suffix, string.length - suffix.length ) !== -1;
+        }
+        
+        function isObj( obj ) {
+        	return obj !== undefined && obj !== null; 
+        }
+        
+        function showRightBlock( delay ) {
+        	$( '#left-block' ).css( 'width', '50%' );
+    		$timeout( function() {
+    			$( '#right-block' ).css( 'visibility', 'visible' );
+    			$( '#right-block' ).css( 'position', 'static' );
+    			$( '#right-block' ).css( 'opacity', '1.0' );
+    		}, delay );
+        }
+        
+        function hideRightBlock() {
+        	$( '#right-block' ).css( 'visibility', 'hidden' );
+    		$( '#right-block' ).css( 'opacity', '0' );
+    		
+    		$timeout( function() {
+    			$( '#left-block' ).css( 'width', '100%' );
+    			$( '#right-block' ).css( 'position', 'absolute' );
+    		}, 1000 );
+        }
+        
+        function sortInstances( instances ) {
```

Comment:
The function was written more than a year ago.
I just "moved" it. And I indeed have to write tests (**it was tested by hand** last year - don't laff  :tomato: ).